### PR TITLE
NO-ISSUE: 0.32.0 CHANGELOGs

### PR DESCRIPTION
**Pull request by paulovmr**

### CODE REVIEW

File: packages/vscode-extension-kogito-bundle/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
+# 0.32.0 (alpha)
+
+- NO-ISSUE - Improved localization for Stunner BPMN Editor
+- NO-ISSUE - Add DMN 1.5 specification files to `packages/dmn-marshaller`
+- [kie-issues#180](https://github.com/kiegroup/kie-issues/issues/180) - On DMN Editor's Boxed Expression Editor, when resizing Context/Invocation entry info cells, minWidths are getting messy
+- [kie-issues#510](https://github.com/kiegroup/kie-issues/issues/510) - Content assist in DMN Editor doesn't allow true/false without major changeskiegroup/kie-tools/pull/1950
```
Comment (caponetto) [inline packages/vscode-extension-kogito-bundle/CHANGELOG.md]: @paulovmr can you please review these lines? Looks like there are some unwanted content in the end of some lines

File: packages/serverless-workflow-vscode-extension/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
+# 0.32.0 (alpha)
+
+- [KOGITO-9668](https://issues.redhat.com/browse/KOGITO-9668) - SWF Editor - In VSCode the diagram reloads if a user clicks on any area of the canvaskiegroup/kie-tools/pull/1886
```
Comment (caponetto) [inline packages/serverless-workflow-vscode-extension/CHANGELOG.md]: @paulovmr can you please review these lines? Looks like there are some unwanted content in the end of some lines

File: packages/dmn-vscode-extension/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,13 @@
+# 0.32.0 (alpha)
+
+- NO-ISSUE - Add DMN 1.5 specification files to `packages/dmn-marshaller`
+- [kie-issues#180](https://github.com/kiegroup/kie-issues/issues/180) - On DMN Editor's Boxed Expression Editor, when resizing Context/Invocation entry info cells, minWidths are getting messy
+- [kie-issues#510](https://github.com/kiegroup/kie-issues/issues/510) - Content assist in DMN Editor doesn't allow true/false without major changeskiegroup/kie-tools/pull/1950
```
Comment (caponetto) [inline packages/dmn-vscode-extension/CHANGELOG.md]: Here too.

File: packages/vscode-extension-kie-ba-bundle/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,14 @@
+# 0.32.0 (alpha)
+
+- NO-ISSUE - Improved localization for Stunner BPMN Editor
+- NO-ISSUE - Add DMN 1.5 specification files to `packages/dmn-marshaller`
+- [kie-issues#180](https://github.com/kiegroup/kie-issues/issues/180) - On DMN Editor's Boxed Expression Editor, when resizing Context/Invocation entry info cells, minWidths are getting messy
+- [kie-issues#510](https://github.com/kiegroup/kie-issues/issues/510) - Content assist in DMN Editor doesn't allow true/false without major changeskiegroup/kie-tools/pull/1950
+- [kie-issues#420](https://github.com/kiegroup/kie-issues/issues/420) - Boxed expression component context menu renders both selected and the chosen cell menu
+- [kie-issues#159](https://github.com/kiegroup/kie-issues/issues/159) - Colorize variable names in FEEL inputs on DMN Editor's Boxed Expression Editorkiegroup/kie-tools/pull/1869
+- [kie-issues#589](https://github.com/kiegroup/kie-issues/issues/589) - Monaco-editor doesn't apply colorisation in DMN Editor1986
```
Comment (caponetto) [inline packages/vscode-extension-kie-ba-bundle/CHANGELOG.md]: Here too.

**Comment by paulovmr** [comment]
@caponetto Thanks, fixed all cases.

**Review by caponetto — APPROVED** [review]

**Review by yesamer — APPROVED** [review]

**Pull request merged**